pygeppetto/data_model.py

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
It's related to what we are discussing, getter and setter are bad practice in Python.

When I started to write the port, I actually rewrote the code for it to be Pythonic and to remove/change some parts.